en-US/Design.xml

```diff
@@ -721,6 +721,34 @@ $ vi myFile.txt
 		</para>
 
 	</section>
+
+	<section id="making-recommendations">
+	 <title>Making Recommendations</title>
+		<para>
+		When making a recommendation, the preferred verbiage is <emphasis role="bold">"Red&nbsp;Hat recommends..."</emphasis> instead of the common but indirect "It is recommended...".
+		Recommendations can include best practices, recommended practices, and product-specific suggestions.
+		Refer to <xref linkend="avoiding-confusing-language"/> for information on using the terms "best practices" and "recommended practices" in Red&nbsp;Hat documentation.
+	  </para>
+    <para>
+	   <emphasis role="strong">Example 1</emphasis>
+	  </para>
+		<para>
+			Incorrect: "Although the attack surface is reduced to the same-project traffic, it is recommended to create multiple service accounts within a project."
+		</para>
+		<para>
+			Correct: "Although the attack surface is reduced to the same-project traffic, Red&nbsp;Hat recommends creating multiple service accounts within a project."
+		</para>
+	 	<para>
+		 <emphasis role="strong">Example 2</emphasis>
```

Comment:
Same comment as before. Also remember to use `&nbsp;` with "Red Hat."

Reply:
Fixed